### PR TITLE
New Portal

### PR DESCRIPTION
**Pull request by zfox23**
Yayyy, it's a new portal script that accepts relative paths!

### CODE REVIEW

File: usefulUtilities/portal/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Portal
+When this script is attached to a zone, the zone serves as a portal, which can be used to transport avatars between domains and within domains.
+
+## Setup
+1. Rez a zone entity that you want to act as the portal.
```
Comment (MiladNazeri) [inline usefulUtilities/portal/README.md]: remove the period in the end to match the others

File: usefulUtilities/portal/portal.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            var userData;
+
+            try {
+                userData = JSON.parse(properties.userData);
+            } catch (e) {
+                console.error("Error parsing userData: ", e);
+            }
+
+            if (userData) {
+                if (userData.destination) {
```
Comment (MiladNazeri) [inline usefulUtilities/portal/portal.js]: Do you need to do a check here to make sure it's a valid destination?  Not sure what happens if you put something wonky in.

Comment (zfox23) [reply usefulUtilities/portal/portal.js]: Nope, we have no way to validate that. We're depending on the script user to supply a valid destination.

File: usefulUtilities/portal/README.md (hunk 1 above)
Comment (zfox23) [reply usefulUtilities/portal/README.md]: all set

**Pull request merged**